lib/block/transaction.go

```diff
@@ -190,19 +190,36 @@ func (bt *BlockTransaction) SaveBlockOperations(st *storage.LevelDBBackend) (err
 	}
 
 	for _, op := range bt.Transaction().B.Operations {
-		var bo BlockOperation
-		bo, err = NewBlockOperationFromOperation(op, bt.Transaction(), bt.blockHeight)
-		if err != nil {
+		if err = bt.SaveBlockOperation(st, op); err != nil {
 			return
 		}
-		if err = bo.Save(st); err != nil {
+	}
+
+	return nil
+}
+
+func (bt *BlockTransaction) SaveBlockOperation(st *storage.LevelDBBackend, op operation.Operation) (err error) {
+	if bt.blockHeight < 1 {
```

Comment:
@spikeekips This part is included `other bugs`, right?

Reply:
@Charleslee522 No, I separated `SaveBlockOperation` from `SaveBlockOperations`, so this part is from `SaveBlockOperations`.

Reply:
@spikeekips Thanks :)